stephanie-smith/short-response.md

```diff
@@ -0,0 +1,66 @@
+# Problem Set 2.2 - Variables and Control Flow
+## Short Response Questions
+
+1. **What does the code below log? Why?**
+   ```javascript
+    {
+      let singer = 'Omar Apollo';
+    }
+
+    console.log(`My favorite singer is ${singer}`);
+   ```
+   - It logs "My favorite singer is Omar Apollo".
+
+2. **What happens when we run the following code? Why?**
+   ```javascript
+   const favorite = 'Juan Pablo';
+   console.log(`Our favorite Marcy Lab family member is ${favorite}!`);
+
+   favorite = 'Maya';
+   console.log(`Actually, ${favorite} is my favorite.`);
+   ```
+   - When you run the code, you get an error about the type used for the variable `favorite`, which is a constant. This means they cannot be redefined.
+
+3. **What does the following code log? Why?**
+   ```javascript
+   let score = 90; 
+   let newScore = score;
+   score = 100; 
+   console.log(score, newScore);
+   ```
+   - It logs 100 and 90. The variable `score` starts off as 90, while a new variable `newScore` is defined as `score`. This means both `score` and `newScore` equal 90. Then, `score` is redefined as 100. The value in `newScore` does not change to 100 despite the newly assigned value, because of the way the code cascades.
```

Comment:
Changes:
* instead of "starts off as 90", say it is _initialized_ with a value of 90.
* instead of "defined as `score`", let's say something like _it is assigned a the value referenced by `score`, which was`90`_
* once again, not redefined but reassigned.
* Lastly... a big one. _cascading_ applies in CSS, not here. The reason that `newScore` still has a value of `90` is because we never reassigned it. We only reassigned `score`.